ipol_demo/modules/core/core.py

```diff
@@ -768,24 +768,21 @@ def copy_blobset_from_physical_location(self, demo_id, work_dir, blobset_id):
         returns:
         """
         try:
-            demo_blobs, _ = self.post(
-                "api/blobs/get_blobs", "post", data={"demo_id": demo_id}
-            )
-            demo_blobs = demo_blobs.json()
+            demo_blobs, status = self.post(f"api/blobs/demo_blobs/{demo_id}", "get")
         except json.JSONDecodeError as e:
             self.logger.exception(
-                f"Blobs get_blobs didn't return valid json at Core's copy_blobset_from_physical_location. {e}"
+                f"Blobs didn't return valid json at Core's copy_blobset_from_physical_location. {e}"
             )
             raise IPOLCopyBlobsError("Couldn't reach blobs")
```

Comment:
This message in the exception is misleading. Actually it could reach Blobs, but it was unable to decode the answer. Let's simply put the text of the exception in a variable and use it to log and the raise it in the next line.